combat/Combat.gd

```diff
@@ -201,6 +201,9 @@ func _process(delta):
 		ui_health.text = str(creature.get_health()) + " / " + str(creature.get_max_health())
 		var ui_ticks = get_node("CanvasLayer/CombatMenu/Allies/VBoxContainer/Ally" + str(i) + "/Ticks")
 		ui_ticks.value = creature.get_ticks()
+		if !creature.is_alive():
+			ui_name.add_color_override("font_color", Color(1,0,0,1))
```

Comment:
Let's turn `Color(1,0,0,1)` into a constant variable and get rid of the magic Color.